docs/docs/contributions/releases/release-strategy.mdx

```diff
@@ -44,6 +44,17 @@ We try our best to write bug-free code, but, like everyone, we sometimes make mi
 If you encounter a bug, please gently let us know by opening a GitHub issue or messaging us on Slack. See [Community](/community/members).
 :::
 
+### Stable release managers
+
+Our weekly [release process](./release-process.mdx) is executed by a rotating "maintainer of the week" (MOTW). But because the entire stable release process (through `dev`s, to an `a`, past `rc`s, etc) can take 8 to 12 weeks, we additionally assign a release manager per stable release to drive the manual steps of a release that are not covered by the weekly process.
+
+The release manager for a stable release is a maintainer chosen informally in Slack to run the upcoming release. They create or are assigned a stable release ticket ([example](https://github.com/pantsbuild/pants/issues/20578)). A stable release has some additional requirements, most of which are listed in the weekly release process steps, but which can require time outside of the weekly process:
+1. Writing (or [automating!](https://github.com/pantsbuild/pants/discussions/19247)) the creation of a release blog ([example](https://www.pantsbuild.org/blog/2024/03/27/pants-2-20))
+2. Tracking/triaging issues which block finalizing the release, in [its milestone](https://github.com/pantsbuild/pants/milestones).
```

Comment:
I think an additional important step here is deciding when _not_ to merge cherry-picks too.

For instance https://github.com/pantsbuild/pants/pull/20779 was a cherry-pick for what seemed like a "non-critical" bug fix (not a regression since 2.19) so we decided to wait for 2.20.0 to go out, rather than merge it and need another RC cycle.

Thus:

```suggestion
2. Tracking/triaging issues which block finalizing the release, in [its milestone](https://github.com/pantsbuild/pants/milestones). This should include rejecting or postponing non-blocking bug-fixes and cherry-picks, if they might delay finalising a release inappropriately.
```


I'm not sure this suggestion quite captures the nuance though.